packages/dataviews/src/stories/dataviews.story.tsx

```diff
@@ -549,3 +554,68 @@ export const InfiniteScroll = () => {
 		</>
 	);
 };
+
+const TimelineComponent = ( {
+	showMedia = 'true',
+}: {
+	showMedia: 'true' | 'false';
+} ) => {
+	const [ view, setView ] = useState< View >( {
+		type: LAYOUT_TIMELINE,
+		search: '',
+		page: 1,
+		perPage: 20,
+		filters: [],
+		fields: [ 'categories', 'datetime' ],
+		titleField: 'title',
+		descriptionField: 'description',
+		groupByField: 'date',
+		mediaField: 'image',
+		showMedia: showMedia === 'true',
+		layout: {
+			eventField: 'date',
+			dateFormat: 'M j, Y',
+		},
+	} );
+	useEffect( () => {
+		const setShowMedia = showMedia === 'true';
+		if ( setShowMedia !== view.showMedia ) {
+			setView( {
+				...view,
+				showMedia: setShowMedia,
+			} );
+		}
+	}, [ view, showMedia ] );
+	const { data: shownData, paginationInfo } = useMemo( () => {
+		return filterSortAndPaginate( data, view, fields );
+	}, [ view ] );
+	return (
+		<DataViews
+			getItemId={ ( item ) => item.id.toString() }
+			paginationInfo={ paginationInfo }
+			data={ shownData }
+			view={ view }
+			fields={ fields }
+			onChangeView={ setView }
+			actions={ actions }
+			defaultLayouts={ {
+				[ LAYOUT_TIMELINE ]: {},
+			} }
+		/>
+	);
+};
+
+export const Timeline = {
```

Comment:
We don't have stories for different layouts, but it may be worth having them if there are specific modes/controls we want to highlight that is difficult to do in the default story. From what I see, this only adds the "media" control and that's something the Default story would benefit as well. So I wonder if we can consolidate there?

Additionally, could you update any other existing story to use this layout as well (update the defaultLayouts const)?